src/components/Tooltip/Tooltip.tsx

```diff
@@ -47,11 +42,18 @@ export default function Tooltip({
     ref: child.ref,
   })
 
-  const transitions = useTransition(
-    isVisible ? tooltipParams : null,
-    null,
-    transitionConfig
-  )
+  const transitions = useTransition(isVisible ? tooltipParams : null, null, {
+    from: { opacity: 0, transform: "translateY(0px)" },
+    enter: {
+      opacity: 1,
+      transform: `translateY(${position === "top" ? `-4px` : `4px`})`,
+    },
+    leave: { opacity: 0, transform: "translateY(0px)" },
```

Comment:
Using translate3d should enable GPU acceleration which you should use when using transitions to lower redraws.

```suggestion
  const transitions = useTransition(isVisible ? tooltipParams : null, null, {
    from: { opacity: 0, transform: "translate3d(0, 0, 0)" },
    enter: {
      opacity: 1,
      transform: `translate3d(${position === "top" ? `0, -4px, 0` : `0, 4px, 0`})`,
    },
    leave: { opacity: 0, transform: "translate3d(0, 0, 0)" },
```

Reply:
Hm, applying this breaks the transform (opacity still working): https://deploy-preview-468--gatsby-interface.netlify.app/?path=/story/tooltip--with-accessible-label … taking a quick look.

Reply:
Resolved in a31454e :v: